test/framework/service.go

```diff
@@ -17,11 +17,12 @@ package framework
 import (
 	"context"
 	"fmt"
-	"k8s.io/apimachinery/pkg/util/yaml"
 	"time"
 
+	"k8s.io/apimachinery/pkg/util/yaml"
+
```

Comment:
Nit: lets remove the new line, at least I don't see the need for it?

Reply:
Whops, not sure how that got there.